checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -94,6 +96,17 @@ protected boolean checkConstructorInvocation(
         return true;
     }
 
+    @Override
+    protected void checkConstructorResult(
+            AnnotatedExecutableType constructorType, ExecutableElement constructorElement) {
+        // Nothing to check
+    }
+
+    @Override
+    protected void checkSuperConstructorCall(MethodInvocationTree superCall) {
```

Comment:
This should be `checkThisOrSuperConstructorCall`.

Reply:
Good catch.  I fixed this:
https://github.com/typetools/checker-framework/commit/f5979aff85ec8f52c96674bcbe2bf58e973b3bec